charts/pega/values-large.yaml

```diff
@@ -83,9 +83,11 @@ global:
         port: 80
         targetPort: 8080
 
+
       replicas: 1
+      # Pass the constellation url as a Dynamic System Setting to the infinity. Added only when Constellation is enabled.
+      nodeSettings: "PEGA-UIENGINE/CONSTELLATIONSVCURL=/prweb/constellation"
```

Comment:
This should probably be passed elsewhere.  Also, there is no need to specify the url since the chart configures it.

Reply:
Please let us know what is the best place to set this DSS.

Reply:
+ @pega-embate

Reply:
I'd suggest adding an entry for the PEGA-UIENGINE/CONSTELLATIONSVCURL param in prweb.xml in the pega-ready image with a value set to a new environment variable that can be set per tier in values.yaml (e.g. CONSTELLATION_URL).

Reply:
Based on today's discussion regarding the constellation architecture, it became apparent that the constellation service URL must be available from outside the Kubernetes cluster (e.g. from a browser) and due to variations in the constellation service deployment, this must be configurable.

To support this, I'd still propose my suggestion of adding an entry for  PEGA-UIENGINE/CONSTELLATIONSVCURL in prweb.xml (this requires an update to the docker pega-web ready image) with a customizable value based on an environment variable specified in values.yaml.

Reply:
PRWeb.xml file: https://github.com/pegasystems/docker-pega-web-ready/blob/master/tomcat-conf/Catalina/localhost/prweb.xml

Add something like:
```
  <Environment name="prconfig/constellation/url" value="{{ .Env.CONSTELLATION_URL }}" type="java.lang.String"/>
```